src/ResourceManager/ServiceFabric/Commands.ServiceFabric/Commands/AddAzureRmServiceFabricApplicationCertificate.cs

```diff
@@ -18,8 +18,8 @@
 using System.Threading.Tasks;
 using Microsoft.Azure.Commands.ServiceFabric.Common;
 using Microsoft.Azure.Commands.ServiceFabric.Models;
-using Microsoft.Azure.Management.Compute;
 using ServiceFabricProperties = Microsoft.Azure.Commands.ServiceFabric.Properties;
+using Microsoft.Azure.Commands.Common.Compute.Version2016_04_preview;
```

Comment:
@praries880 This compute dependency seems old. We need to use the latest API version for compute calls.

Reply:
@juhacket Thanks for the review Justin.
Which would be a good version of Compute for you to go to :
 https://github.com/Azure/azure-rest-api-specs/tree/master/specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01
or
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/compute/resource-manager/Microsoft.Compute/stable/2018-04-01